tests_e2e/templates/execute-tests.yml

```diff
@@ -33,4 +33,22 @@ jobs:
           # Add all KeyVault secrets explicitly as they're not added by default to the environment vars
           AZURE_CLIENT_ID: $(AZURE-CLIENT-ID)
           AZURE_CLIENT_SECRET: $(AZURE-CLIENT-SECRET)
-          AZURE_TENANT_ID: $(AZURE-TENANT-ID)
\ No newline at end of file
+          AZURE_TENANT_ID: $(AZURE-TENANT-ID)
+
+      - task: CopyFiles@2
+        inputs:
+          sourceFolder: $(HOME)/tmp
+          contents: '**/*junit.xml'
+          targetFolder: $(Build.ArtifactStagingDirectory)/harvest
+        displayName: 'Copy test results'
+
+      - task: PublishTestResults@2
+        inputs:
+          testResultsFormat: 'JUnit'
+          testResultsFiles: '**/*junit.xml'
+          searchFolder: $(Build.ArtifactStagingDirectory)/harvest
+          testRunTitle: 'Publish test results'
+
+      - publish: $(Build.ArtifactStagingDirectory)/harvest
+        artifact: 'test-harvest'
```

Comment:
This is temp name, needs to change to have unique name per run(distro name+scenario name etcc)